pkgs/by-name/sx/sxcs/package.nix

```diff
@@ -0,0 +1,46 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitea,
+  xorg
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  name = "sxcs";
+  version = "1.1.0";
+
+  src = fetchFromGitea {
+    domain = "codeberg.org";
+    owner = "NRK";
+    repo = "sxcs";
+    rev = "refs/tags/v${finalAttrs.version}";
+    hash = "sha256-rYmbbdZjeLCvGvNocI3+KVU2KBkYvRisayTyScTRay8=";
+  };
+
+  buildInputs = [ xorg.libX11 xorg.libXcursor ];
+
+  buildPhase = ''
+    runHook preBuild
+    ${stdenv.cc.targetPrefix}cc -o sxcs sxcs.c -O3 -s -l X11 -l Xcursor
+    runHook postBuild
+  '';
+
+  outputs = [ "out" "man" ];
+
+  installPhase = ''
+    runHook preInstall
+
+    install -Dm 755 sxcs $out/bin/sxcs
```

Comment:
```suggestion
    install -Dm755 sxcs -t $out/bin
```

Reply:
Just learned about the `-t` option, pretty nice to avoid repeating binary name :o